src/api/utils/audio-over-socket.js

```diff
@@ -73,16 +73,16 @@ export function registerStreamForRecorder(recorder, rpcName) {
     const {audioParameters: {channels, sampleRate}} = recorder.getAudioSpecs();
     const headerArrBuff = createWAVEHeader(channels, sampleRate);
     const header = Array.from(new Uint8Array(headerArrBuff));
-    let headerSent = false;
+    let sentHeader = true;
```

Comment:
I see the use of sentHeader in paused, but the meaning is reversed now. is sentHeader == true I expect the header to be sent. Perhaps you should rename to sendHeader to reflect that you still need to do it

Reply:
My rationale for changing this:
`sentHeader === true` means: send the header. Sending it wil set it to false (as you can see). When a pause occurs, it wil be set to true again. This made more sense to me than what the reverse did. Then I would have to set `headerSent` to false at pause, which is semantically not true because I did sent the header, but I need to send it again.

What I do have to change (as I see now) is that I made a stupid mistake (imo) on `sentHeader`. I should have called it `sendHeader` (present tense) 😛 

That might actually be what you mean with this review ;-)